definitions/modules/autopilot.yaml

```diff
@@ -1,134 +1,275 @@
+    var_name: A:AIRSPEED INDICATED
     var_units: Knots
-    var_type: i32
+    var_type: f64
     event_name: AP_SPD_VAR_SET
-  # Do we need AUTOPILOT MACH HOLD?
-  -
+
+  - # ASOBO_AUTOPILOT_Push_VerticalSpeed_Template
     type: ToggleSwitch
     var_name: A:AUTOPILOT VERTICAL HOLD
     var_units: Bool
     event_name: AP_PANEL_VS_HOLD
-  - # Doesn't need a toggleswitch, implicitly climbs aircraft to set altitude
+
+  - # ASOBO_AUTOPILOT_Knob_VerticalSpeed_Template
+    type: NumSet
+    var_name: A:AUTOPILOT AIRSPEED HOLD VAR
+    var_units: Knots
+    var_type: f64
+    event_name: AP_SPD_VAR_SET
+  -
+    type: event
+    event_name: AP_AIRSPEED_ON
+  -
+    type: event
+    event_name: AP_SPD_VAR_INC
+  -
+    type: event
+    event_name: AP_SPD_VAR_DEC
+  -
+    type: event
+    event_name: AP_MACH_ON
+  -
+    type: event
+    event_name: AP_MACH_VAR_INC
+  -
+    type: event
+    event_name: AP_MACH_VAR_DEC
+  -
     type: NumSet
     var_name: A:AUTOPILOT VERTICAL HOLD VAR
-    var_units: Feet/minute
+    var_units: Feet per minute
     var_type: i32
     event_name: AP_VS_VAR_SET_ENGLISH
   -
+    type: event
+    event_name: AP_VS_VAR_INC
+  -
+    type: event
+    event_name: AP_VS_VAR_DEC
+  -
+    type: var
+    var_name: L:XMLVAR_SpeedIsManuallySet
+    var_type: f64
+  -
+    type: var
+    var_name: L:XMLVAR_AirSpeedIsInMach
+    var_type: f64
+
+  - # ASOBO_AUTOPILOT_Knob_SpeedMach_Template
+    type: ToggleSwitch
+    var_name: A:AUTOPILOT MANAGED SPEED IN MACH
+    var_units: Bool
+    event_name: AP_MANAGED_SPEED_IN_MACH_TOGGLE
+
+  - # ASOBO_AUTOPILOT_Push_FlightLevelChange_Template
     type: ToggleSwitch
     var_name: A:AUTOPILOT FLIGHT LEVEL CHANGE
     var_units: Bool
     event_name: FLIGHT_LEVEL_CHANGE
-  -
+
+  - # ASOBO_AUTOPILOT_Push_Level_Template
     type: ToggleSwitch
     var_name: A:AUTOPILOT WING LEVELER
     var_units: Bool
-    var_type: bool
-    event_name: AP_WING_LEVELER_ON
-    off_event_name: AP_WING_LEVELER_OFF
+    event_name: AP_PITCH_LEVELER_ON
+    off_event_name: AP_PITCH_LEVELER_OFF
   -
     type: ToggleSwitch
-    var_name: A:AUTOPILOT NAV1 LOCK
+    var_name: A:AUTOPILOT WING LEVELER
     var_units: Bool
-    var_type: bool
-    event_name: AP_NAV1_HOLD
-  -
+    event_name: AP_WING_LEVELER_ON
+    off_event_name: AP_WING_LEVELER_OFF
+
+  - # ASOBO_AUTOPILOT_Push_BackCourse_Template
     type: ToggleSwitch
     var_name: A:AUTOPILOT BACKCOURSE HOLD
     var_units: Bool
-    var_type: bool
     event_name: AP_BC_HOLD
   -
     type: ToggleSwitch
     var_name: A:AUTOPILOT PITCH HOLD
     var_units: Bool
-    var_type: bool
     event_name: AP_ATT_HOLD_ON
     off_event_name: AP_ATT_HOLD_OFF
-  -
+
+  - # ASOBO_AUTOPILOT_Push_FlightDirector_Template
     type: ToggleSwitch
     var_name: A:AUTOPILOT FLIGHT DIRECTOR ACTIVE
     var_units: Bool
-    var_type: bool
     event_name: TOGGLE_FLIGHT_DIRECTOR
   -
+    type: ToggleSwitch
+    var_name: A:AUTOPILOT FLIGHT DIRECTOR ACTIVE:1
```

Comment:
probably should get rid of the definition above if you're going to use these ones with indexes